modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -572,6 +584,21 @@ public function clear_auth_cookies() {
 		// Post password cookie.
 		setcookie( 'wp-postpass_' . COOKIEHASH, ' ', time() - YEAR_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN );
 	}
+
+	/**
+	 * Activates the Classic Editor plugin and reloads the page so it can initialize.
+	 *
+	 * This will only work if the Classic Editor plugin is already installed.
+	 */
+	public function activate_classic_editor() {
+		// phpcs:ignore WordPress.Security.NonceVerification
+		if ( ! empty( $_GET['set-editor'] ) && 'classic' === $_GET['set-editor'] && current_user_can( 'activate_plugin' ) ) {
+			if ( is_plugin_inactive( 'classic-editor/classic-editor.php' ) ) {
```

Comment:
Do we need to account for the case where we end up here and the plugin is installed and active?

Reply:
If the plugin is already installed and active, we might wish to reset the plugin's options to be using the classic editor if the user's previously switched them to the block editor.

Reply:
The reason for the `is_plugin_inactive` check is to see if the plugin is installed....